### PR TITLE
Making cfripper compatible with Python3.10

### PR DESCRIPTION
**Pull request by w0rmr1d3r**
Relates to https://github.com/Skyscanner/cfripper/issues/218

### Improvements
 - CFRipper is now compatible with Python3.10
 ### Updates
 - Bump dev dependency `moto` to allow `>=3.0.0`.

### CODE REVIEW

**Review by ocrawford555 — APPROVED** [review]
Future-proofing woo!

File: setup.py [hunk 1]
```diff
@@ -16,14 +16,17 @@
     "PyYAML>=4.2b1",
 ]
 
+# docker is needed to use moto 3.0.0 and above
+# see https://github.com/spulec/moto/issues/3567#issuecomment-947047013
 dev_requires = [
     "black==22.3.0",
+    "docker>=5.0.0",
```
Comment (oscarbc96) [inline setup.py:23]: This requirement should be installed by moto

File: setup.py [hunk 2]
```diff
@@ -16,14 +16,17 @@
...
     "flake8>=3.3.0",
     "isort==4.3.21",
     "pytest>=3.6",
     "pytest-cov>=2.5.1",
     "pip-tools>=5.3.1",
-    "moto==1.3.14",
+    "moto>=3.0.0",
```
Comment (oscarbc96) [inline setup.py:29]: Let's use the extra requirements provided by moto
```suggestion
    "moto[cloudformation,s3]>=3.0.0",
```

**Review by oscarbc96 — CHANGES_REQUESTED** [review]

**Review by jsoucheiron — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (w0rmr1d3r) [reply setup.py]: true, got it deleted in [12062ed](https://github.com/Skyscanner/cfripper/pull/219/commits/12062ed53e819209ee3c996a7134aadf0738a5ec)

File: setup.py (hunk 2 above)
Comment (w0rmr1d3r) [reply setup.py]: done in [12062ed](https://github.com/Skyscanner/cfripper/pull/219/commits/12062ed53e819209ee3c996a7134aadf0738a5ec)

**Review by oscarbc96 — APPROVED** [review]

**Review by jsoucheiron — APPROVED** [review]

**Review by oscarbc96 — APPROVED** [review]

**Pull request merged**